metals/src/main/scala/scala/meta/internal/metals/MetalsLspService.scala

```diff
@@ -1260,68 +896,13 @@ class MetalsLspService(
           renameProvider.runSave(),
           parseTrees(path),
           onChange(List(path)),
-        ) ++ // if we fixed the script, we might need to retry connection
-          maybeImportScript(
```

Comment:
Yes, this method is overriden in `ProjectMetalsLspService` and this part is not needed for fallback, since there will be no tree view for it currently, and we already load all files in fallback as scripts.